src/gRPC/hpo_pb2.py

```diff
@@ -14,133 +13,10 @@
-  })
-_sym_db.RegisterMessage(NumberExperimentsParams)
-
-ExperimentsListParams = _reflection.GeneratedProtocolMessageType('ExperimentsListParams', (_message.Message,), {
-  'DESCRIPTOR' : _EXPERIMENTSLISTPARAMS,
-  '__module__' : 'hpo_pb2'
-  # @@protoc_insertion_point(class_scope:helloworld.ExperimentsListParams)
-  })
-_sym_db.RegisterMessage(ExperimentsListParams)
-
-ExperimentEmptyReply = _reflection.GeneratedProtocolMessageType('ExperimentEmptyReply', (_message.Message,), {
-  'DESCRIPTOR' : _EXPERIMENTEMPTYREPLY,
-  '__module__' : 'hpo_pb2'
-  # @@protoc_insertion_point(class_scope:helloworld.ExperimentEmptyReply)
-  })
-_sym_db.RegisterMessage(ExperimentEmptyReply)
-
-ExperimentNameParams = _reflection.GeneratedProtocolMessageType('ExperimentNameParams', (_message.Message,), {
-  'DESCRIPTOR' : _EXPERIMENTNAMEPARAMS,
-  '__module__' : 'hpo_pb2'
-  # @@protoc_insertion_point(class_scope:helloworld.ExperimentNameParams)
-  })
-_sym_db.RegisterMessage(ExperimentNameParams)
-
-ExperimentTrial = _reflection.GeneratedProtocolMessageType('ExperimentTrial', (_message.Message,), {
-  'DESCRIPTOR' : _EXPERIMENTTRIAL,
-  '__module__' : 'hpo_pb2'
-  # @@protoc_insertion_point(class_scope:helloworld.ExperimentTrial)
-  })
-_sym_db.RegisterMessage(ExperimentTrial)
-
-ExperimentTrialResult = _reflection.GeneratedProtocolMessageType('ExperimentTrialResult', (_message.Message,), {
-  'DESCRIPTOR' : _EXPERIMENTTRIALRESULT,
-  '__module__' : 'hpo_pb2'
-  # @@protoc_insertion_point(class_scope:helloworld.ExperimentTrialResult)
-  })
-_sym_db.RegisterMessage(ExperimentTrialResult)
-
-ExperimentDetails = _reflection.GeneratedProtocolMessageType('ExperimentDetails', (_message.Message,), {
-
-  'Tunable' : _reflection.GeneratedProtocolMessageType('Tunable', (_message.Message,), {
-    'DESCRIPTOR' : _EXPERIMENTDETAILS_TUNABLE,
-    '__module__' : 'hpo_pb2'
-    # @@protoc_insertion_point(class_scope:helloworld.ExperimentDetails.Tunable)
-    })
-  ,
-  'DESCRIPTOR' : _EXPERIMENTDETAILS,
-  '__module__' : 'hpo_pb2'
-  # @@protoc_insertion_point(class_scope:helloworld.ExperimentDetails)
-  })
-_sym_db.RegisterMessage(ExperimentDetails)
-_sym_db.RegisterMessage(ExperimentDetails.Tunable)
-
-TunableConfig = _reflection.GeneratedProtocolMessageType('TunableConfig', (_message.Message,), {
-  'DESCRIPTOR' : _TUNABLECONFIG,
-  '__module__' : 'hpo_pb2'
-  # @@protoc_insertion_point(class_scope:helloworld.TunableConfig)
-  })
-_sym_db.RegisterMessage(TunableConfig)
-
-TrialConfig = _reflection.GeneratedProtocolMessageType('TrialConfig', (_message.Message,), {
-  'DESCRIPTOR' : _TRIALCONFIG,
-  '__module__' : 'hpo_pb2'
-  # @@protoc_insertion_point(class_scope:helloworld.TrialConfig)
-  })
-_sym_db.RegisterMessage(TrialConfig)
-
-_HPOSERVICE = DESCRIPTOR.services_by_name['HpoService']
+_builder.BuildMessageAndEnumDescriptors(DESCRIPTOR, globals())
+_builder.BuildTopDescriptorsAndMessages(DESCRIPTOR, 'hpo_pb2', globals())
 if _descriptor._USE_C_DESCRIPTORS == False:
```

Comment:
Was this file automatically generated with `grpc_tools.protoc `? it appears to have removed a lot of the protobuf configuration

Reply:
Yes. It was generated automatically. With the updates to protobuf version, these changes were seen.